components/editor/modules/embed/EmbedLoader.js

```diff
@@ -70,6 +70,10 @@ export default (query, Component) => (
       const active = editor.value.blocks.some(
         block => block.key === node.key
       )
+
+      const data = node.data.toJS()
+      // We need to guard against undefined thumbnail below which happens
```

Comment:
Note that I still need this even with
https://github.com/orbiting/styleguide/commit/3590bfd53efdd0e3ffa5e01660e086ff9ca01204